### DISCUSSION TITLE
[ FF7 ] Graphic Corruption / Flickering on AMD GPUs

### DISCUSSION THREAD

**Discussion by DropsMJay** [julianxhokaxhiu/FFNx#299] [Q&A] [answered]
**Bug description**
When i use the DirectX11 renderer, the game have graphics corruption/flickering. 

**To Reproduce**
Just play the game.
Video: https://www.youtube.com/watch?v=SGO40sL5gE4

**Computer specs:**

AMD FX 3,3Ghz Six Core
AMD RX 550 4GB - Driver version: 21.10.2
8GB RAM

**Comment by eve-atum** [comment, marked as answer]
1.9.0.0 is six months out of date, and AMD is known to be cranky with D11.  Update to canary (which just went feature-lock earlier tonight), and check your RX drivers (which also seem to be just a hair behind)

**Comment by malvarenga123** [comment]
I have the same issue with 1.10.0.0 (also with AMD video card updated to latest drivers). Changing the renderer to Vulkan fixes it.
